docs/source/overview/index.rst

```diff
@@ -0,0 +1,53 @@
+Overview (for Programmers)
+===================================
+
+This page provides a broad overview of how the model works. It is intended for programmers who want to read and/or edit the code.
+
+.. highlight:: bash
+
+
+Source Code and Packages
+---------------------------------
+
+The main package for the source code is ``microsim`` and the entry-point for the program is the ``main.py`` file (i.e. ``microsim/main.py``). To run the model and see what arguments are available, execute the following from the root project directory:
```

Comment:
at the end of this line if you add ` ::` it will render the indented block as preformatted code (just to keep the styling consistent)

Reply:
Nice, thanks, I'll update the file